packages/web3-core-promievent/package.json

```diff
@@ -8,6 +8,6 @@
     "main": "src/index.js",
     "dependencies": {
         "any-promise": "1.3.0",
-        "eventemitter3": "1.1.1"
+        "eventemitter3": "3.1.2"
```

Comment:
I'm a bit worried about bumping a major version on a dependency, as it could introduce potential incompatibilities in something as critical as the promievent. Why was this needed?